src/cpx_env.jl

```diff
@@ -50,6 +50,9 @@ function close_CPLEX(env::Env)
     end
 end
 
+#= CPLEX API's used in this function are not longer supported and was deprecated since 12.8
```

Comment:
@odow this is the code to fix the breaking change.

Reply:
Oops, I scrolled right past this. But this will now break earlier versions? What minimum version of CPLEX should we support?

Reply:
This change will make the minimum version supported 12.8 

https://www.ibm.com/support/knowledgecenter/en/SSSA5P_12.8.0/ilog.odms.studio.help/CPLEX/ReleaseNotes/topics/releasenotes128/deprecated.html

There is no mention of this change in the release notes for 12.71 
https://www.ibm.com/support/knowledgecenter/en/SSSA5P_12.7.1/ilog.odms.studio.help/CPLEX/ReleaseNotes/topics/releasenotes1271/deprecated.html

Reply:
If so, we should update
https://github.com/JuliaOpt/CPLEX.jl/blob/bb3f45c236fb0afb2fa2ed75c448e17b9a64a61e/deps/build.jl#L26

Reply:
Actually, we should move the current master to a branch `pre-1280` and drop the support for versions older than 12.80 in master.

Reply:
I think it's reasonable to remove pre 12.8 support. If people complain, they can volunteer to maintain the code.

Reply:
Ok, will also update the README file